rhizome/inference_endpoint/lib/replica_setup.rb

```diff
@@ -18,8 +18,8 @@ def prep(inference_engine:, inference_engine_params:, model:, replica_ubid:, ssl
   end
 
   def start_systemd_units
-    r "systemctl start lb-cert-download.timer"
-    r "systemctl start inference-engine.service"
+    r "systemctl enable --now lb-cert-download.timer"
+    r "systemctl enable --now inference-engine.service"
```

Comment:
I didn't know about the `--now` flag for `enable`. I used to run `start` and `enable` separately. That's neat 🚀